app/screens/SearchLibrary/components/Header/styles.js

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+import { white } from '@scss/colors';
+
+export const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    height: 35,
+    marginTop: 40,
+    marginHorizontal: 10,
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    backgroundColor: white,
+    borderRadius: 20
+  },
+  backgroundImage: {
+    marginTop: -20,
+    width: '100%',
+    height: 110
+  },
+  iconLeft: {
+    height: 15,
+    width: 15,
+    marginLeft: 10
+  },
+  iconRight: {
+    height: 15,
+    width: 15,
```

Comment:
since is the same as before extract this in a style called `icon : { height: ICON_SIZE, width: ICON_SIZE }`